rpmlint/checks/BinariesCheck.py

```diff
@@ -25,6 +25,48 @@ def create_nonlibc_regexp_call(call):
     r = r'.*?\s+UND\s+(%s)\s?.*$' % call
     return re.compile(r)
 
+class Issue:
+    """
+    A simple issues wrapper that later reported via add_info.
+    """
+
+    def __init__(self, level, reason, description = None):
+        self.level = level
+        self.reason = reason
+        self.description = description
+
+class BinaryInfoChecker:
```

Comment:
I wonder if this must be separate function from BinaryCheck

Reply:
I tend to come up with the class. Mainly because I want it to test and I don't want to mock up `class BinaryInfo(object)` which has a complex constructor:

```
    def __init__(self, config, output, pkg, path, fname, is_ar, is_shlib):
        self.path = path
        self.readelf_error = False
...
```